### PR TITLE
GUI: KBFS revoke endangered TLFs warning

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

screenshot from 2016-11-03 16-44-14

revkbfs1

There's also a merge of `device-revoke/dumb.{desktop,native}.js`, since they were almost identical.

### CODE REVIEW

**Comment by chrisnojima** [comment]
Shouldn't it be left aligned / have bullets?  
screen shot 2016-11-03 at 5 30 25 pm

File: shared/actions/devices.js [hunk 1]
```diff
@@ -41,10 +44,25 @@ export function removeDevice (deviceID: string, name: string, currentDevice: boo
...
+
 export function generatePaperKey (): GeneratePaperKey {
   return {type: Constants.generatePaperKey, payload: undefined}
 }
 
+function * _deviceShowRemovePageSaga (showRemovePageAction: ShowRemovePage): SagaGenerator<any, any> {
+  const device = showRemovePageAction.payload.device
+  let endangeredTLFs = {endangeredTLFs: []}
+  try {
+    endangeredTLFs = yield call(rekeyGetRevokeWarningRpcPromise, {param: {targetDevice: device.deviceID}})
```
Comment (chrisnojima) [inline shared/actions/devices.js]: hmmm. I'm a little worried about this call taking awhile and the user being confused. This is fine short term but in general i think we should either
1. Do some waiting state (either make the button you clicked waiting or something else, cursor maybe? if you click a row)
or
2. Actually go to the new page and have a spinner and do this loading when the component routes.

let's discuss during scrum

**Comment by chrisnojima** [comment]
Code is fine, see 💬  above.  We should fixup the styling to match

**Comment by cjb** [comment]
Switched to styling matching Zeplin in 8346bb8:

screenshot from 2016-11-04 11-33-11

**Pull request merged**